File: template.R
#!/usr/bin/env Rscript

library("argparse")

parser <- ArgumentParser(description="")
parser$add_argument("file", help="input data tsv file")
parser$add_argument("-o", "--output", help="output prefix")
args <- parser$parse_args()

if(!is.null(args$output)) { out <- args$out } else {
    out <- sub("\\..*", "", args$file)
}

data <- read.table(args$file, header=T, sep="\t")



library("ggplot2")

pdf(file=paste0(out, ".pdf"))
p1 <-

dev.off()

File: template.py
#!/usr/bin/env python3

"""
@author Yi Chen
@date
"""

import argparse

if __name__ == "__main__":
    parser = argparse.ArgumentParser(description="""Description""")
    parser.add_argument("-v", "--val", type=int, default=1, metavar="n", help="value")
    params = parser.parse_args()
